daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -798,7 +803,7 @@ private[lf] object Speedy {
             .initial(pkg2TxVersion, contractKeyUniqueness, submissionTime, initialSeeding),
           committers = committers,
           readAs = readAs,
-          commitLocation = None,
+          commitLocation = commitLocation,
```

Comment:
Nice :+1:

Reply:
I checked why this didn’t blow up before and it turns out we didn’t actually use commitLocation for anything. It was originally used before I split scenario interpretation into off-ledger and on-ledger interpretation but with the split we had access to it in `ScenarioRunner` already.

So at least nothing else was broken by the fact that we didn’t set this.

Reply:
Ok, good ... I was a bit worried.